.azure-pipelines/test_plan.py

```diff
@@ -377,14 +372,6 @@ def poll(self, test_plan_id, interval=60, timeout=36000, expected_states=""):
         dest="interval",
         help="Polling interval. Default 60 seconds."
     )
-    parser_poll.add_argument(
```

Comment:
Don't remove the timeout parameter here and let default as -1, which stands for timeout not enabled.
And let it be compatible no matter if the timeout is passed or not.